src/app/tests/suites/commands/interaction_model/InteractionModel.cpp

```diff
@@ -125,6 +125,12 @@ void InteractionModel::OnSubscriptionEstablished(SubscriptionId subscriptionId)
     ContinueOnChipMainThread(CHIP_NO_ERROR);
 }
 
+void InteractionModel::OnResubscriptionAttempt(CHIP_ERROR aTerminationCause, uint32_t aNextResubscribeIntervalMsec)
+{
+    ChipLogError(chipTool, "\nwould trigger re-subscription");
+    ContinueOnChipMainThread(CHIP_NO_ERROR);
```

Comment:
I don't think we should be doing `ContinueOnChipMainThread` here.  Why would we do that?

Reply:
removed, thanks